executor/cleanup.go

```diff
@@ -65,26 +69,14 @@ func cleanup(client *kubernetes.Clientset, namespace string, instanceId string)
 	if err != nil {
 		return err
 	}
-	// See K8s #33845 and related bugs: deleting a deployment
-	// through the API doesn't cause the associated ReplicaSet to
-	// be deleted.  (Fixed recently, but we may be running a
-	// version before the fix.)
-	err = cleanupReplicaSets(client, namespace, instanceId)
-	if err != nil {
```

Comment:
I am sorry but why did we remove the error check here?

Reply:
The bug described in the comment was fixed in k8s 1.6. Since fission required 1.7+ , remove the legacy code here.

Reply:
I guess you mean L84, fixed.